packages/server/lib/browsers/chrome.js

```diff
@@ -118,7 +118,11 @@ const pluginsBeforeBrowserLaunch = function (browser, args) {
   })
 }
 
-const _normalizeArgExtensions = function (dest, args) {
+const _normalizeArgExtensions = function (dest, args, browser) {
```

Comment:
can we have some JSDocs here? or comments if the browser is headless, then we cannot load any extensions, thus we return arguments as is. Because it is unclear what this function does - if the browser is headless we return args - but what if the arguments include `LOAD_EXTENSION`? shouldn't we filter them out?

Reply:
if the plugin returns extensions, they are filtered out elsewhere. in any case, chrome will still successfully launch with `--load-extension` passed, but no extension will really be loaded

Reply:
converted the .js to .ts and added documentation: 86969ee